locallibrary/catalog/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
```

Comment:
Chưa có khai báo gì thì viết gọn lại em nhé
```suggestion
urlpatterns = []
```

Reply:
Dạ